### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by rubensa**
Fix https://github.com/spring-projects/spring-security/issues/9315 for Spanish

### CODE REVIEW

File: web/src/main/resources/org/springframework/security/web/messages_es.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+ExceptionTranslationFilter.insufficientAuthentication=Para acceder a este recurso se requiere autenticaci\u00f3n completa
```
Comment (sjohnr) [inline web/src/main/resources/org/springframework/security/web/messages_es.properties:1]: Please also add this translation to `core/src/main/resources/org/springframework/security/messages_es_ES.properties`.

**Review by sjohnr — CHANGES_REQUESTED** [review]
Hi @rubensa. Thanks for this submission! Please see my feedback inline.

Also, will you please squash your commits and format the resulting commit like so:

```
Commit Title

Closes gh-9315
```

**Comment by rubensa** [comment]
@sjohnr Done!  Sorry for the delay.

**Comment by rubensa** [comment]
@sjohnr Just realized that `PersistentTokenBasedRememberMeServices.cookieStolen` in `core/src/main/resources/org/springframework/security/messages_es_ES.properties` was in English so I translated it too.

**Pull request merged**